File: src/main/kotlin/aris/thesis/theratricalplaysapi/repositories/SystemRepository.java
package aris.thesis.theratricalplaysapi.repositories;

import aris.thesis.theratricalplaysapi.entities.System;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.data.jpa.repository.JpaSpecificationExecutor;

public interface SystemRepository extends JpaRepository<System, Integer>, JpaSpecificationExecutor<System> {

}

File: src/main/kotlin/aris/thesis/theratricalplaysapi/repositories/SystemRepository.kt
package aris.thesis.theratricalplaysapi.repositories

import aris.thesis.theratricalplaysapi.entities.System
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.data.jpa.repository.JpaSpecificationExecutor
import org.springframework.stereotype.Repository

@Repository
interface SystemRepository : JpaRepository<System, Int>